python/py-nbval/Portfile

```diff
@@ -4,16 +4,10 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        computationalmodelling nbval 0.9.6
+github.setup        computationalmodelling nbval 0.11.0
```

Comment:
please add `github.tarball_from archive` ; that will likely require you to update the `checksums` as well

Reply:
Thanks done.